system/loggerd/tests/test_logger.cc

```diff
@@ -97,7 +97,6 @@ TEST_CASE("logger") {
 
     auto logging_thread = [&]() -> void {
       LoggerHandle *lh = logger_get_handle(&logger);
-      REQUIRE(lh != nullptr);
```

Comment:
catch2 is not thread safe

Reply:
fixed: https://github.com/commaai/openpilot/pull/29852